WorkingAtBabylon.md

```diff
@@ -0,0 +1,98 @@
+<img src="logo.png">
+</p>
+
+
+Working in the iOS team
+==================================
+
+This document is modelled after Robin Malhotra's [Questions to ask your iOS interviewer](https://github.com/codeOfRobin/Questions-to-ask-your-interviewer-iOS). It aims to provide a few answers to how we work at Babylon and our process is structured.
+
+### Where do we store our code?
+We use Github. We have a central repo that hosts the main and white label apps, and a [few Open Source projects](https://github.com/search?q=topic%3Aios+org%3ABabylonpartners+fork%3Atrue) that we created and rely on.
+
+## Continuous Integration
+### What Continuous Integration service do you use?
+
+We currently run on CircleCI
+
+### How many apps/targets/schemes?
+
+We have a main app and a few white-label apps.
+
+We use a modular architecture, so our app workspace is split into 9 Xcode projects for modules and apps, 3 projects for test utilities and shared features, and 4 projects related to our SDK, not counting the Pods project (and our OSS projects)
+
+Each module project has 3 targets on average: one for the framework, one for the Unit Test and one for Snapshot tests when applicable. The app project has 18 targets (6*3) given our white-labelling variants.
+### How big is your Fastfile?
+It's around 250 lines of code at the moment, with about 15-20 lanes for beta testing, daily and custom builds, etc.
+
+### Does it have several environment references?
+### Can you dry run a lane on your machine right now?
+Yes.
+
+### Codesigning?
+  
+* What architectural patterns are you using to make development easier or more consistent? What problems with your current approach have you come across after settling on one (if you have one)?
+
+## Team/People
+### comms - slack?
+Slack, as a non-trivial part of the team is remote. Zoom for meetings of more than 2-3 people.
+
+### Do people ever take sick days to get work done?
+No.
+
+### When was the last time someone worked a late night/weekend in order to meet a deadline?
+TODO
+
+### How many meetings do you have in a day?
+It depend on the squad you’re assigned. For Enrolment and Integrity I counted 18 in a typical sprint, so that averages to ~2 a day. Note that quite a few of them are optional.
+
+Interviews, sometimes, can take up some time. It's usually, at maximum, one hour per week.
+
+### Does your calendar look like a losing game of tetris? Do you have large continuous uninterrupted hours?
+It depends a lot on the squad. For Enrolment and Integrity, for example, most meetings are on the first Monday and Tuesday of the sprint. For Triage and Health Management this is not the case and are more spread out.
+
+### Are there any private spaces in your office?
+There are very few at the moment.
+
+### Is there a gym/field/court close to your office for physical activity?
+There is a commercial gym close by, a park, and a barebones gym in the office. There’s also free yoga twice a week and a football club
```

Comment:
```suggestion
There is a commercial gym close by, a park, and a barebones gym in the office. There’s also free yoga twice a week and a weekly 5-a-side football game.
```
We don't play against other companies (yet) so I'm not sure we could call it a "club" :p